src/sagas/wallet.js

```diff
@@ -194,7 +194,8 @@ export function* startWallet(action) {
     connection = wallet.conn;
   } else {
     // Set the server and network as saved on localStorage
-    config.setServerUrl(LOCAL_STORE.getServer());
+    // If the localStorage is empty, fetch data directly from the lib config
+    config.setServerUrl(LOCAL_STORE.getServer() || config.getServerUrl());
```

Comment:
issue(blocking): If thats the case we should update the storage as well.

Since closing the app will load from localStorage we need to keep the localStorage updated with the current network.

Reply:
I'm not sure this would be necessary, as the `server_url` configurations were correct right after wallet load. However, I think it's a good practice to update both the local store and the lib configuration together at all times until we unify them somehow.

Done on [2634e07](https://github.com/HathorNetwork/hathor-wallet/pull/537/commits/2634e070cf9675ec78a06528e36a11fa1d31b920)